src/app/shared/components/time-bar/time-bar.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
```

Comment:
La requête getSunRiseSet est lancée toutes les 10 sec...

Reply:
OK, let's set 10 seconds interval between requests.

Reply:
What I meant is that this request should probably not be launched every 10 sec.